### ISSUE TITLE
jipdate complains "no file provided and not in query mode" even though I provided a file with -f

### ISSUE THREAD

**Issue by pm215** [Linaro/jipdate#21]
```
e104462:xenial:jipdate$ EDITOR=/bin/true JIRA_USERNAME=peter.maydell@linaro.org ~/linaro/jipdate/jipdate.py -f ~/jipdate.txt
No file provided and not in query mode

usage: jipdate.py [-h] [-e] [-f FILE] [-l] [-p] [-q] [-s] [-t] [-u USER] [-v]
                  [-x] [--all] [--dry-run]

[etc]
```
This worked with commit 741de62a0cf4d54, but has broken somewhere between there and commit 044808e59572.

I tried to do a bisect, but in commit 87ebeb2934c70fdab it can't even print the error message and instead you get a python backtrace:
```
e104462:xenial:jipdate$ EDITOR=/bin/true JIRA_USERNAME=peter.maydell@linaro.org ~/linaro/jipdate/jipdate.py -f ~/jipdate.txt
Traceback (most recent call last):
  File "/home/petmay01/linaro/jipdate/jipdate.py", line 499, in <module>
    main(sys.argv)
  File "/home/petmay01/linaro/jipdate/jipdate.py", line 467, in main
    eprint("No file provided and not in query mode\n")
  File "/home/petmay01/linaro/jipdate/helper.py", line 7, in eprint
    print(*args, file=sys.stderr, **kwargs)
NameError: global name 'sys' is not defined
```

**Comment by jbech-linaro** [comment]
Hi Peter,

Alright, I'll have a look. Must have went in with my latest patches. I couldn't get anyone to test them, so after 2-3 weeks I just merged it.

**Comment by jbech-linaro** [comment]
Hmm, two things, first I think [this](https://github.com/Linaro/jipdate/blob/master/jipdate.py#L425) check isn't correct, there should be a **not** for the file argument also (I'll submit a patch for that).

Secondly, I cannot reproduce the error, I've tried combinations with `-f foo.txt`, `-q -f foo.txt` and I do not get the python backtrace, in fact it seems to work as it should.

I cannot understand why you get
```python
NameError: global name 'sys' is not defined
```

Since in `helper.py`, there is a [line](https://github.com/Linaro/jipdate/blob/master/helper.py#L2) saying
```python
import sys
```

I've tested this on Ubuntu 16.04 and on Arch Linux (both using Python2.x) and it is working. Could it be some package missing?

**Comment by jbech-linaro** [comment]
I've pushed the patch for the argument issue at least. I can give it a try in a docker container to see if I can reproduce it on a clean `xenial`.

**Comment by jbech-linaro** [comment]
Tested it on clean `xenial` (using Docker image ubuntu16.04) also and it works fine.

```bash
root@92117924d1c2:/jipdate# EDITOR=/bin/true JIRA_USERNAME=joakim.bech@linaro.org /jipdate/jipdate.py -f /test.txt -t
Password: 
These JIRA cards will be updated as follows:

[SWG-226]
  Adding another comment

Server to update: TEST
 https://dev-projects.linaro.org

Are you sure you want to update Jira with the information above? [y/n] y
Successfully updated your Jira tickets!

This is your status:

---

# Subject: [Weekly] Week ending 2017-10-13

Hi,

This is the status update from me for the last week.

Cheers!


Joakim Bech


[SWG-226]
# Header: ATSHA204a mezzanine support
# Type: Epic
# Status: In Progress
# No updates since last week.
Adding another comment

[SWG-18]
# Header: GlobalPlatform adaptations
# Type: Initiative
# Status: In Progress
# No updates since last week.

[SWG-1]
# Header: TEE/TrustZone: Open Source Trusted OS
# Type: Initiative
# Status: In Progress
# No updates since last week.

root@92117924d1c2:/jipdate#
```

**Comment by pm215** [comment]
Thanks -- just confirmed that this is fixed by commit 30e157385cec3e5fedcabb8c8e1a4c9ee67a8fae.
